documentation/book/con-upgrade-intro.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='con-upgrade-intro-{context}']
```

Comment:
I think concept modules should include a heading.

Reply:
I've rearranged.